test/notifications/display_test.dart

```diff
@@ -166,7 +188,7 @@ void main() {
       final stream = eg.stream();
       final message = eg.streamMessage(stream: stream);
       await checkNotifications(async, messageFcmMessage(message, streamName: null),
-        expectedTitle: '(unknown stream) > ${message.subject}',
+        expectedTitle: '#(unknown stream) > ${message.subject}',
```

Comment:
Do you want to extract this out to the other commit on stream renaming? I'm guessing it isn't necessary for this commit to work?

Reply:
@sumanthvrao Sorry but I am not sure what's requested here, this change is already in a separate commit and it is the first one in the sequence. Would you want this commit to be after the main commit?

Also this change is fixing #572 which is just adding a `#` before stream names, the `streams` -> `channel` renaming is being done in https://github.com/zulip/zulip-flutter/pull/707 which includes changes for these notification strings.

Reply:
Oh, apologies if I am misunderstanding here, but how are the two changes in your `summary notif commit` that add a # before stream name different from your first commit that does the same (for 2 other strings in different part of the codebase)? I was suggesting you merge the former into the first commit?

Reply:
Ah, sorry for the misunderstanding, this should definitely belong in the first commit.